docker/constraints.in

```diff
@@ -17,7 +17,7 @@ jsonschema
 lark-parser>=0.6.7
 matplotlib
 moto
-netcdf4
+netcdf4>=1.5.8
```

Comment:
When editing this file you really need to regenerate `constraints.txt` and not just manually edit it, since you are not compiling netcdf4 but getting binary wheels from pip you need to ensure that numpy version is compatible with that BINARY version of netcdf4.

Reply:
Actually scratch that. netcdf4 doesn't have a wheel, it's compiled inside the docker, but clearly against the wrong numpy version. so probably geobase problem, when building wheels one needs to ensure that numpy you build against and numpy you run against are compatible.